tests/fixtures/post-release/.sauce/config_mac.yml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1alpha
+kind: testcafe
+sauce:
+  region: us-west-1
+  concurrency: 10
+  metadata:
+    tags:
+      - e2e
+      - post release
+    build: Github Run $GITHUB_RUN_ID
+testcafe:
+  # CAUTION: This has to be an already deployed framework version in the cloud. Not necessarily the one you want to test.
+  # Then use --runner-version to specify the release you actually want to use.
+  version: 1.14.2
+rootDir: ./
+
+#####################
+# All the mac tests #
+#####################
+suites:
+  - name: "test safari on mac"
```

Comment:
If you're bringing back mac tests, may as well add at least one ios test

Reply:
That has been added @mhan83 !